jenkins-jobs/pipelines/release/release-charts-pipeline.groovy

```diff
@@ -0,0 +1,189 @@
+import groovy.json.*
+import java.util.stream.*
+
+import com.cloudbees.groovy.cps.NonCPS
+
+@Library("dbz-libs") _
+
+if (
+    !RELEASE_VERSION ||
+    !DEBEZIUM_OPERATOR_REPOSITORY ||
+    !DEBEZIUM_OPERATOR_BRANCH ||
+    !DEBEZIUM_PLATFORM_REPOSITORY ||
+    !DEBEZIUM_PLATFORM_BRANCH ||
+    !OCI_ARTIFACT_REPO_URL
+) {
+    error 'Input parameters not provided'
+}
+
+GIT_CREDENTIALS_ID = 'debezium-github'
+QUAYIO_CREDENTIALS_ID = 'debezium-charts-quay'
+HOME_DIR = '/home/cloud-user'
+GPG_DIR = 'gpg'
+
+RELEASE_SEM_VERSION=RELEASE_VERSION.replaceAll(/\.(?=[^.]*$)/, '-').toLowerCase()
```

Comment:
Add a comment why is this needed

Reply:
This is mostly related to https://github.com/debezium/debezium/pull/6183. I'll do the changes on that branch

Reply:
Addressed on the relative PR.